css/cssom/CSS-namespace-object-class-string.any.js

```diff
@@ -0,0 +1,40 @@
+"use strict";
+// https://heycam.github.io/webidl/#es-namespaces
+// https://drafts.csswg.org/cssom/#namespacedef-css
+
+test(() => {
+  assert_own_property(CSS, Symbol.toStringTag);
```

Comment:
`typeof CSS` is `"function"` in WebKit and Blink, but an `"object"` in Gecko.
Would you consider aligning this Gecko if I add a test and submit WebKit patch?

Reply:
In WebKit, `CSS` is defined as regular WebIDL interface rather than _namespace object_, causing this behavior.
Fixing it would require introducing _namespace object_ concept to WebIDL's bindings generator.

Reply:
Yes, we plan to implement CSS as a namespace in Chromium; we have the same legacy issue.